runtime/autoload/health/provider.vim

```diff
@@ -254,11 +254,17 @@ function! s:check_python(version) abort
   if !empty(pyenv)
     if empty(pyenv_root)
       call health#report_warn(
-            \ 'pyenv was found, but $PYENV_ROOT is not set.',
-            \ ['Did you follow the final install instructions?',
-            \  'If you use a shell "framework" like Prezto or Oh My Zsh, try without.',
-            \  'Try a different shell (bash).']
```

Comment:
Aren't these suggestions still valid?

Reply:
It feels a little bit misleading for me because using framework / using different shells seems to be not related here, considering that if `$PYENV_ROOT` is not explicitly set, pyenv will use the default `~/.pyenv` anyway.